docs/features/dataset-usage-and-query-history.md

```diff
@@ -0,0 +1,75 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Dataset Usage & Query History
+
+<FeatureAvailability/>
+
+Dataset Usage & Query History can give dataset-level information about the top queries which referenced a dataset.
```

Comment:
It's a bit odd because the Title and the opening sentence references "Dataset" usage in particular, but then you go on to mention "Chart" and "Dashboard" usage.

Can we keep it consistent and maybe just either a) focus on datasets and omit anything about charts and dashboards or b) broaden the purpose of this doc to include ingesting usage stats about dashboards and charts